kbs/src/attestation/backend.rs

```diff
@@ -45,6 +45,15 @@ lazy_static! {
 
 pub type TeeEvidence = serde_json::Value;
 
+#[derive(Deserialize)]
+pub struct KbsEvidence {
+    /// The TEE evidence.
+    evidence: String,
+
+    /// The init data is optional. Note that it is initdata plaintext.
+    init_data_toml: Option<String>,
+}
```

Comment:
Not sure this struct is really necessary, especially after our discussion about putting the init-data in the protocol itself rather than in the evidence.